components/wmtk_components/embedded_component/internal/EmbeddedRemeshing.hpp

```diff
@@ -0,0 +1,46 @@
+#pragma once
+
+#include <igl/adjacency_list.h>
+#include <igl/edges.h>
+#include <igl/triangle/triangulate.h>
+#include <wmtk/Scheduler.hpp>
+#include <wmtk/TriMesh.hpp>
+
+namespace wmtk::components::internal {
+
+class EmbeddedRemeshing2D
+{
+public:
+    EmbeddedRemeshing2D(
+        Eigen::MatrixXi& m_edges_,
+        Eigen::MatrixXd& m_vertices_,
+        double m_blank_rate_ = 0.5
+        // double m_resolute_area = 0.1
+    );
+    void compute_bounding_value(double& max_x, double& max_y, double& min_x, double& min_y);
+    void process();
+
+    Eigen::MatrixXi m_edges;
+    Eigen::MatrixXd m_vertices;
+    // m_blank_rate is used to define how big the boundary should be
+    // if the m_blank_rate = 0.5, then the bounding box's x and y length
+    // should be respectively 0.5 larger than the range of x and y of input
+    // Therefore, the area of the bounding box is as 4 times as the input's bounding box's.
```

Comment:
Just say that each side of the bbox will be double the input size in that case. Talking about area here is a bit confusing.